packages/db/prisma/schema.prisma

```diff
@@ -57,6 +58,13 @@ model Absence {
   @@unique([absenceDate, userId])
 }
 
+model PushToken {
+  token   String  @id
```

Comment:
I think it makes more sense to mark this field as @unique -- they are essentially the same but logically I think unique makes more sense